sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -1329,8 +1329,7 @@ private[client] class Shim_v3_0 extends Shim_v2_3 {
       isSrcLocal: Boolean): Unit = {
     val session = SparkSession.getActiveSession
     assert(session.nonEmpty)
```

Comment:
In this case, it seems that we can remove the above two lines together, doesn't it?

Reply:
Ah!! sorry, I missed that will raise follow-up

Reply:
I raised the follow-up PR  [29736](https://github.com/apache/spark/pull/29736). Sorry for the trouble